frontend/src/api/client.ts

```diff
@@ -0,0 +1,9 @@
+import createFetchClient from "openapi-fetch";
+import createClient from "openapi-react-query";
+import type { paths } from "./types";
+
+const fetchClient = createFetchClient<paths>({
+  baseUrl: "http://localhost:8080",
```

Comment:
This needs to be configurable and in production we will probably want it to be a subpath.